Kitodo/src/main/java/org/kitodo/services/data/TaskService.java

```diff
@@ -87,67 +88,8 @@ public static TaskService getInstance() {
     }
 
     /**
-     * Creates and returns a query to retrieve tasks for which the currently logged
```

Comment:
This part shouldn't be removed.

Reply:
I had unsaved editor tabs during the rebase. This caused the loss. This happend due to the time pressure I was under to make a hot fix. It is very good that you found it, and I have put the code back into good condition.

GitHub does not update this part of the review because the line you made the comment on was wrapped differently by my code formatter. Click “files changed” register on top and navigate to `TaskService.java` to see that it is actually fixed.